test_work/run_mig.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+VER=1.15.pre
+DBCFG=../database.yml.local
+SEEDSQL=../mdummy_test.sql
+cd wagn/
+gem build wagn.gemspec
+mv *.gem ../card/
+cd ../decko-rails
+gem build decko.gemspec
+mv *.gem ../card/
+cd ../card
+gem build card.gemspec
+gem install *-${VER}.gem
```

Comment:
Mainly for dev purposes so I don't have to re-install the gems when working on them.  Also wanted to test that code path.

The point is more that we can do it a number of ways if we want, and add other scenarios as their own scripts.  I added the whole directory to .gitignore with that thought.  We'll only want to add stuff to the repo explicitly and ignore them most of the time.